### DISCUSSION TITLE
[Q] Vertical order info

### DISCUSSION THREAD

**Discussion by ialexalvarado** [exceljs/exceljs#2370] [Q&A]
Is there a way to have order the information something like this?
<img width="410" alt="Screen Shot 2022-06-21 at 21 14 33" src="https://user-images.githubusercontent.com/29547899/174928835-abc1a58f-3d62-4141-b346-6c0844bef7dc.png">

I took a look in all documentation but i don't see something explicit for this, i mean, as i understand you can only build your spreadsheet in this way

<img width="346" alt="Screen Shot 2022-06-21 at 21 21 26" src="https://user-images.githubusercontent.com/29547899/174929521-a939b168-5cff-4101-b7ed-00fd443e2656.png">

Regards

**Comment by mani-cafs** [comment]
@ialexalvarado  Yes you can create like first picture but you have to change the array structure ` var workbook     = new Exceljs.Workbook(); const worksheet = workbook.addWorksheet(sheetname); var  result_info    = []; result_info[0]       = ["Name","Willian","Peter"] result_info[1] = ["Phone",834859434,9343523423], result_info[2] = ["City","New York","Los Angeles"] result_info.forEach((final_val)=>{worksheet .addRow(final)}) const buf =  await workbook.xlsx.writeBuffer();saveAs(new Blob([buf]), `${"work_sheet"}.xlsx`);                                                                                                                       
 `
